modules/programs/thunderbird.nix

```diff
@@ -7,6 +7,8 @@ let
 
   cfg = config.programs.thunderbird;
 
+  jsonFormat = pkgs.formats.json { };
```

Comment:
this could have been `thunderbirdJson` including the description to avoid the repetition.

Reply:
Done